### PR TITLE
Add contribution templates + minor doc improvements

### PR DESCRIPTION
**Pull request by Ma27**
This is just a proposal, I think it's helpful for first-time contributors to provide them a template which "tells" them what kind of information is needed.

### CODE REVIEW

**Comment by Ma27** [comment]
fixed, train wifi seems to work today %)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Type of PR
+
+* [ ] Bugfix
+* [ ] New Feature
+* [ ] Breaking change
```
Comment (mmoll) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think "Breaking Change" should be on a separate Line/Section and the third point should be "something else" or "other".

Comment (Ma27) [reply .github/PULL_REQUEST_TEMPLATE.md]: Regarding the addition of "else": full ack :+1: 

Regarding the "Breaking change" box: how would you propose to name this section?

I certainly get your point, I wasn't sure about this either, but in the end I thought that nobody forbids you to check multiple boxes (these are intentionally check boxes, not radio boxes). But as I said, I'm not sure about this and happy to discuss :)

**Comment by mmoll** [comment]
merged, danke @Ma27!

**Pull request merged**